DEPS

```diff
@@ -91,6 +91,115 @@ vars = {
 
   # Setup Git hooks by default.
   "setup_githooks": True,
+
+  # upstream URLs for third party dependencies, used in
```

Comment:
Please add instructions here that explain how to extend this list when a new dependency is added. Is there some way that people check whether the upstream repo URL is recognized by the vuln scanning service before landing a change?